new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItServerStartPolicy.java

```diff
@@ -0,0 +1,1237 @@
+ *   IF_NEEDED->ALWAYS->IF_NEEDED
+ *
+ * <p>testStartAlwaysDynamicClusterManaged
+ *  Restart a server in dynamic cluster (beyond replica count) 
+ *   IF_NEEDED->ALWAYS->IF_NEEDED
+ *
+ * <p>testStopNeverConfigClusterManaged
+ *  Stop a server in configured cluster by changing serverStartPolicy
+ *   IF_NEEDED->ALWAYS->NEVER
+ *
+ * <p>testStopNeverDynamicClusterManaged
+ *  Stop a server in dynamic cluster by changing serverStartPolicy
+ *   IF_NEEDED->ALWAYS->NEVER
+ *
+ * <p>testStopConfigClusterReplicaManaged
+ *  Change the serverStartPolicy of a running managed server (say ms1) in config
+ *  cluster to NEVER. 
+ *  Make sure next managed server (say ms2) is scheduled to run to mantain the 
+ *  replica count whle the running managed server ms1 goes down.
+ *  Change the serverStartPolicy of server ms1 to IF_NEEDED.
+ *  Make sure server ms2 goes down and server ms1 is re-scheduled to mantain 
+ *  the replica count
+ *
+ * <p>testStopDynamicClusterReplicaManaged
+ *  Change the serverStartPolicy of a running managed server (say ms1) in a 
+ *  dynamic cluster to NEVER. 
+ *  Make sure next managed server (say ms2) is scheduled to run to mantain the 
+ *  replica count whle the running managed server ms1 goes down.
+ *  Change the serverStartPolicy of server ms1 to IF_NEEDED.
+ *  Make sure server ms2 goes down and server ms1 is re-scheduled to mantain 
+ *  the replica count
+ *
+ * <p>testStandaloneManagedRestartIfNeeded
+ *  Restart standalone server changing serverStartPolicy 
+ *   IF_NEEDED->NEVER->IF_NEEDED
+ *
+ * <p>testStandaloneManagedRestartAlways
+ *  Restart standalone server changing serverStartPolicy 
+ *   IF_NEEDED->NEVER->ALWAYS
+ */
+
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+@DisplayName("ServerStartPolicy attribute in different scope in a mii domain")
+@IntegrationTest
+class ItServerStartPolicy {
+
+  private static HelmParams opHelmParams = null;
+  private static V1ServiceAccount serviceAccount = null;
+  private String serviceAccountName = null;
+  private static String opNamespace = null;
+  private static String operatorImage = null;
+  private static String domainNamespace = null;
+  private static ConditionFactory withStandardRetryPolicy = null;
+  private static String dockerConfigJson = "";
+
+  private static int replicaCount = 1;
+  private static final String domainUid = "mii-start-policy";
+  private StringBuffer curlString = null;
+
+  private StringBuffer checkCluster = null;
+  private V1Patch patch = null;
+
+  private static Map<String, Object> secretNameMap;
+
+  private final String adminServerPodName = domainUid + "-admin-server";
+  private final String managedServerPrefix = domainUid + "-managed-server";
+  private static LoggingFacade logger = null;
+
+  /**
+   * Install Operator.
+   * Create domain resource defintion.
+   * @param namespaces list of namespaces created by the IntegrationTestWatcher by the
+   JUnit engine parameter resolution mechanism
```

Comment:
Document how many clusters are created and how many servers in it initially.

Reply:
Create a WebLogic domain with one dynamic cluster (with two managed servers)
 one configured cluster (with two managed servers) and a standalone manged 
 server. The replica count is set to 1 and serverStartPolicy is set to 
 IF_NEEDED at managed server level.